subtests/docker_cli/run/run.py

```diff
@@ -73,6 +73,7 @@ def run_once(self):
     def postprocess(self):
         super(run_base, self).postprocess()  # Prints out basic info
         dockercmd = self.sub_stuff['dkrcmd']
+        self.logdebug(dockercmd)
```

Comment:
okay, and remember too the test was only attached to stdout, so we probably were throwing away any errors anyway.